constants.py

```diff
@@ -35,16 +35,16 @@
     'Gasóleo colorido e marcado'
 )
 
-# Hisotry plot
-HISTORY_PLOT_START_DATE = 'start_date'
-HISTORY_PLOT_END_DATE = 'start_date'
+# History plot
 HISTORY_PLOT_LABEL_GASOLINA_IO95 = 'Gasolina IO95'
 HISTORY_PLOT_LABEL_GASOLINA_IO98 = 'Gasolina IO98'
 HISTORY_PLOT_LABEL_GASOLEO_RODOVIARIO = 'Gasoleo Rodoviario'
 HISTORY_PLOT_Y_LABEL = 'Preço (€)'
 HISTORY_PLOT_X_LABEL = 'Data'
 
 # CSV columns
+COLUMN_START_DATE = 'start_date'
+COLUMN_END_DATE = 'start_date'
```

Comment:
Are we using this constant?

Reply:
You are right, we are not using the `COLUMN_END_DATE`, but just for completeness, I would leave it there.

Reply:
Hmm sure, but at least give it a proper value: `end_date`.